WordPress/Classes/Utility/In-App Feedback/AppRatingsUtility.swift

```diff
@@ -11,7 +11,7 @@ class AppRatingUtility: NSObject {
     /// section so as to trigger the review prompt for a fairly active user who
     /// uses the app in a broad fashion.
     ///
-    @objc var systemWideSignificantEventCountRequiredForPrompt: Int = 1
+    @objc var systemWideSignificantEventCountRequiredForPrompt: Int = 5
```

Comment:
This is strange. I pushed a commit to revert this change, so locally the value of this variable is `1`, and yet it shows `5` in the PR.

![CleanShot 2024-01-10 at 14 56 58@2x](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/5bd5f0e7-e074-4db3-a9c6-969ef8c9431c)

But I moved this file to a different folder, which is probably why Git got confused.

Reply:
I'll checkout to your branch and verify